source/user-manual/registering/command-line-registration.rst

```diff
@@ -0,0 +1,154 @@
+.. Copyright (C) 2019 Wazuh, Inc.
+
+.. _command-line-registration:
+
+Registering the Wazuh agents using the command line (CLI)
+=========================================================
+
+This method consists on registering the Wazuh agent with the Wazuh manager using ``manage_agents`` utility by extracting the registration key from the Wazuh manager and inserting it manually in the Wazuh agent.
+
+.. note:: Root/Administrator user privileges are necessary to execute all the commands described below.
+
+Wazuh manager
```

Comment:
Is this subtitle the best possible? What about "Wazuh agent key extraction from the Wazuh manager" or something like that. Wazuh manager suggests... nothing. 
I think that must be reviewed on this page and all others. I know that you want to say: In the Wazuh manager, this must be run. But, the title can be clearer.

Reply:
Done